libs/sdk-core/src/greenlight/node_api.rs

```diff
@@ -2026,6 +2031,78 @@ impl NodeAPI for Greenlight {
+
+        let mut client = self.get_node_client().await?;
+        let res = match node_cmd {
+            NodeCommand::ListPeers => {
+                let req = cln::ListpeersRequest::default();
+                let resp = with_connection_retry!(client.list_peers(req.clone()))
+                    .await?
+                    .into_inner();
+
+                Ok(crate::serializer::value::to_value(&resp)?)
+            }
+            NodeCommand::ListPeerChannels => {
+                let req = cln::ListpeerchannelsRequest::default();
+                let resp = with_connection_retry!(client.list_peer_channels(req.clone()))
+                    .await?
+                    .into_inner();
+                Ok(crate::serializer::value::to_value(&resp)?)
+            }
+            NodeCommand::ListFunds => {
+                let req = cln::ListfundsRequest::default();
+                let resp = with_connection_retry!(client.list_funds(req.clone()))
+                    .await?
+                    .into_inner();
+                Ok(crate::serializer::value::to_value(&resp)?)
+            }
+            NodeCommand::ListPayments => {
+                let req = cln::ListpaysRequest::default();
+                let resp = with_connection_retry!(client.list_pays(req.clone()))
+                    .await?
+                    .into_inner();
+                Ok(crate::serializer::value::to_value(&resp)?)
+            }
+            NodeCommand::ListInvoices => {
+                let req = cln::ListinvoicesRequest::default();
+                let resp = with_connection_retry!(client.list_invoices(req.clone()))
+                    .await?
+                    .into_inner();
+
+                Ok(crate::serializer::value::to_value(&resp)?)
+            }
+            NodeCommand::CloseAllChannels => {
+                let req = cln::ListpeersRequest::default();
+                let resp = with_connection_retry!(client.list_peers(req.clone()))
+                    .await?
+                    .into_inner();
+                for p in resp.peers {
+                    self.close_peer_channels(hex::encode(p.id)).await?;
+                }
+
+                Ok(Value::String("All channels were closed".to_string()))
+            }
+            NodeCommand::GetInfo => {
+                let req = cln::GetinfoRequest::default();
+                let resp = with_connection_retry!(client.getinfo(req.clone()))
+                    .await?
+                    .into_inner();
+                Ok(crate::serializer::value::to_value(&resp)?)
+            }
+            NodeCommand::Stop => {
+                let req = cln::StopRequest::default();
+                let resp = with_connection_retry!(client.stop(req.clone()))
+                    .await?
+                    .into_inner();
+                Ok(crate::serializer::value::to_value(&resp)?)
+            }
+        };
+        Ok(res)
+    }
```

Comment:
These dev commands are already serialized with the custom serializer here: https://github.com/breez/breez-sdk-greenlight/blob/527669a438b5cde3a5e0a18515e284bd944881d2/libs/sdk-core/src/breez_services.rs#L1122

Are you using the dev commands from the node api directly? Otherwise I think this change is not necessary?